packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.test.tsx

```diff
@@ -226,8 +225,8 @@ describe('AllergyForm ', () => {
     await user.type(screen.getByLabelText(/Date of onset and comments/i), comment);
     await user.click(screen.getByRole('button', { name: /save and close/i }));
 
-    expect(mockShowToast).toHaveBeenCalledTimes(1);
-    expect(mockShowToast).toHaveBeenCalledWith({
+    expect(mockShowSnackbar).toHaveBeenCalledTimes(1);
+    expect(mockShowSnackbar).toHaveBeenCalledWith({
       critical: true,
```

Comment:
```suggestion
      isLowContrast: true,
```

Reply:
> 

thanks for the review @denniskigen , updating that immediately. Den please why do we have this CI fail in most cases.

Reply:
You have a failing test

![CleanShot 2024-01-22 at 1  05 01@2x](https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/10f48a6c-0b2a-4dc0-9703-ea44edaf7a10)

Reply:
The `critical` and `description` props are only relevant to `showToast`. You should use `isLowContrast` and `subtitle` in their place, respectively.

Reply:
thanks @denniskigen could not see this in the test file.

Reply:
@denniskigen i noticed, when setting up the project, some files, like yarn.lock, .yarnc.yml have some modifications. I doubt if we are to push including them 
Kindly let me know what to do in that case

Reply:
Generally speaking, you shouldn't see changes to `yarn.lock` unless you install a package. `yarnrc.yml` shouldn't change unless you bump your Yarn version (either locally in the project or globally). I've only seen one other case with the `.yarnrc.yml` changing unexpectedly, and in their case, it required reinstalling Yarn, so YMMV. In this case, I'd recommend resetting your git history, removing changes to those files, and then committing anew.

Reply:
Okay, but this comes after issueing 'yarn install' when setting up the project.

Reply:
What's the result of running `yarn -v` in the Patient Chart repo, and globally, at the root of your PC in your home folder or similar.

Reply:
4.0.2

Reply:
> What's the result of running `yarn -v` in the Patient Chart repo, and globally, at the root of your PC in your home folder or similar.

Hey @denniskigen i have resolve it all tests are passing now